### ISSUE TITLE
Gulp build does not uglify app.js

### ISSUE THREAD

**Issue by MarinAngelo** [angular-fullstack/generator-angular-fullstack#2642]
- [x] I understand that GitHub issues are not for tech support, but for questions specific to this generator, bug reports, and feature requests.

Item | Version
----- | -----
generator-angular-fullstack | 3.7.5
Node | 6.5.0
npm | 3.10.3
Operating System Ubuntu 16.04

Item | Answer
----- | -----
Transpiler | Babel
Markup | HTML
CSS | LESS
Router | ui-router
Client Tests | Mocha
DB | MongoDB
Auth | Y

Hi there, 

Is there a specific reason why gulp build does not uglyfi app.js or ist it an error?

Kind Regards

Marinus

**Comment by Awk34** [comment]
Sounds like a bug to me. Have you changed anything that might affect this?

**Comment by MarinAngelo** [comment]
No, I don't think so, the only thing I have changed in the gulp file is added a copy block: 

`gulp.task('copy:gridfonts', () => {
    return gulp.src([
        `${clientPath}/bower_components/angular-ui-grid/ui-grid.svg`,
        `${clientPath}/bower_components/angular-ui-grid/ui-grid.ttf`,
        `${clientPath}/bower_components/angular-ui-grid/ui-grid.woff`,
        `${clientPath}/bower_components/angular-ui-grid/ui-grid.eot`
    ], { dot: true })
        .pipe(gulp.dest(`${paths.dist}/${clientPath}/app`));
});`

**Comment by MarinAngelo** [comment]
Here is the console output:

`[09:40:55] Requiring external module babel-register
[09:41:02] Using gulpfile m:\Websites\projekte\Kunden\SVS\svs-ngFullStack\gulpfile.babel.js
[09:41:02] Starting 'build'...
[09:41:02] Starting 'clean:dist'...
[09:41:02] Starting 'clean:tmp'...
[09:41:02] Finished 'clean:tmp' after 171 ms
[09:41:02] Finished 'clean:dist' after 178 ms
[09:41:02] Starting 'inject'...
[09:41:02] Starting 'inject:js'...
[09:41:02] Starting 'inject:css'...
[09:41:02] Starting 'inject:less'...
[09:41:02] gulp-inject Nothing to inject into index.html.
[09:41:02] Finished 'inject:css' after 187 ms
[09:41:02] gulp-inject 35 files into app.less.
[09:41:02] Finished 'inject:less' after 249 ms
[09:41:03] gulp-inject 96 files into index.html.
[09:41:03] Finished 'inject:js' after 353 ms
[09:41:03] Finished 'inject' after 354 ms
[09:41:03] Starting 'wiredep:client'...
[09:41:03] Finished 'wiredep:client' after 65 ms
[09:41:03] Starting 'transpile:client'...
[09:41:03] Starting 'transpile:server'...
[09:41:06] Finished 'transpile:server' after 2.99 s
[09:41:06] Finished 'transpile:client' after 3.8 s
[09:41:06] Starting 'build:images'...
[09:41:07] Starting 'copy:extras'...
[09:41:07] Starting 'copy:fonts'...
[09:41:07] Starting 'copy:assets'...
[09:41:07] Starting 'copy:server'...
[09:41:07] Starting 'copy:gridfonts'...
[09:41:07] Starting 'styles'...
[09:41:07] Starting 'html'...
[09:41:07] Starting 'constant'...
[09:41:07] Finished 'constant' after 181 ms
[09:41:07] Finished 'copy:extras' after 384 ms
[09:41:10] Finished 'copy:server' after 2.78 s
[09:41:10] Finished 'copy:gridfonts' after 2.78 s
[09:41:10] gulp-imagemin: Minified 1 image (saved 3.71 kB - 30.1%)
[09:41:10] Finished 'styles' after 2.78 s
[09:41:10] Finished 'build:images' after 3.24 s
[09:41:10] Finished 'copy:assets' after 2.8 s
[09:41:10] Finished 'copy:fonts' after 2.85 s
[09:41:10] Finished 'html' after 2.88 s
[09:41:10] Starting 'build:client'...
[09:41:29] Finished 'build:client' after 20 s
[09:41:29] Finished 'build' after 27 s`

**Comment by Awk34** [comment]
Make sure any monospace code/console output you paste is surrounded by triple backticks (\`\`\`)

**Comment by umimehar** [comment]
hello Marinus,
Can you please show your screenshot of app.js
its same happens to me when i Gulp build. 

<img width="1394" alt="screen shot 2017-10-08 at 03 46 39" src="https://user-images.githubusercontent.com/11488033/31312395-5143a1b8-abdb-11e7-9305-6dba391727d6.png">

and when i run `node server`, it run the express server on 9000 port but only returns the static index page with loading message. looks like angular is not working in dist. 

sorry if i am doing something wrong but this way i am learning. 
Any help from you guys will help me alot. 
@Awk34 
Thanks.

**Comment by MarinAngelo** [comment]
Hi there
I could not resolve this problem jet. app.js in dist does get uglified after gulp build. What can I do?

**Comment by Awk34** [comment]
Do you think you can send me your project code?
